README.md

```diff
@@ -54,9 +54,13 @@ The Ada library is used by important systems besides Node.js such as Redpanda, K
 
 [![the ada library](http://img.youtube.com/vi/tQ-6OWRDsZg/0.jpg)](https://www.youtube.com/watch?v=tQ-6OWRDsZg)<br />
 
-## Quick Start
+## Installation
+
+Binary packages for the following systems are currently available:
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/ada.svg)](https://repology.org/project/ada/versions)
```

Comment:
this is coool!!!

Reply:
we should add it to simdutf and simdjson :-)